frontend/sass/partials/_footer.scss

```diff
@@ -1,12 +1,26 @@
 .footer {
     background-color: #3c3c3c;
+    padding: 30px 0;
 	&_list{
-	    list-style: none;
+	    text-align: center;
 	}
 	&_item{
 	    list-style: none;
+        display: inline;
```

Comment:
could you please tidy up the spacing? Can just use this to help get the spacing all the same across the file: http://www.cleancss.com/css-beautify/